holopy/inference/nmpfit.py

```diff
@@ -165,7 +166,7 @@ def minimize(self, parameters, obj_func):
 
         def resid_wrapper(p, fjac=None):
```

Comment:
I know this is from before your commit, but it would be good if we could change the single-character variable name to something more descriptive.

Reply:
renamed to parameters